<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="0" ignored="0" total="1" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Suite_demo" duration-ms="9588" started-at="2021-06-10T10:30:46 ICT" finished-at="2021-06-10T10:30:55 ICT">
    <groups>
    </groups>
    <test name="Demo_Chrome" duration-ms="9588" started-at="2021-06-10T10:30:46 ICT" finished-at="2021-06-10T10:30:55 ICT">
      <class name="ExercisemvnTestcase.Testcase_login">
        <test-method status="PASS" signature="driver_open(java.lang.String, java.lang.String)[pri:0, instance:ExercisemvnTestcase.Testcase_login@33c7e1bb]" name="driver_open" is-config="true" duration-ms="2393" started-at="2021-06-10T10:30:46 ICT" finished-at="2021-06-10T10:30:48 ICT">
          <params>
            <param index="0">
              <value>
                <![CDATA[Chrome]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Data.xlsx]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- driver_open -->
        <test-method status="PASS" signature="login_error_page()[pri:2, instance:ExercisemvnTestcase.Testcase_login@33c7e1bb]" name="login_error_page" duration-ms="2951" started-at="2021-06-10T10:30:48 ICT" finished-at="2021-06-10T10:30:51 ICT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login_error_page -->
        <test-method status="FAIL" signature="getResult(org.testng.ITestResult)[pri:0, instance:ExercisemvnTestcase.Testcase_login@33c7e1bb]" name="getResult" is-config="true" duration-ms="5" started-at="2021-06-10T10:30:51 ICT" finished-at="2021-06-10T10:30:51 ICT">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=login_error_page status=SUCCESS method=Testcase_login.login_error_page()[pri:2, instance:ExercisemvnTestcase.Testcase_login@33c7e1bb] output={null}]]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.NullPointerException">
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException
at ExerciseBaseCommon.BaseTest.getResult(BaseTest.java:166)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:132)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:61)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:366)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:320)
at org.testng.internal.TestInvoker.runConfigMethods(TestInvoker.java:701)
at org.testng.internal.TestInvoker.runAfterGroupsConfigurations(TestInvoker.java:677)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:661)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:174)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:822)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:147)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:764)
at org.testng.TestRunner.run(TestRunner.java:585)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1218)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
at org.testng.TestNG.runSuites(TestNG.java:1069)
at org.testng.TestNG.run(TestNG.java:1037)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
        <test-method status="PASS" signature="driver_quit()[pri:0, instance:ExercisemvnTestcase.Testcase_login@33c7e1bb]" name="driver_quit" is-config="true" duration-ms="4215" started-at="2021-06-10T10:30:51 ICT" finished-at="2021-06-10T10:30:55 ICT">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- driver_quit -->
      </class> <!-- ExercisemvnTestcase.Testcase_login -->
    </test> <!-- Demo_Chrome -->
  </suite> <!-- Suite_demo -->
</testng-results>
